<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1557391950348" stop="1557391950414">
  <name>scripts</name>
  <labels/>
  <test-cases>
    <test-case start="1557391950348" status="passed" stop="1557391950353">
      <name>C:\Users\dongxin\PycharmProjects\5.9\scripts.testlogin.Testlogin.test_hello_001</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="11308-MainThread"/>
        <label name="host" value="DESKTOP-041HO6S"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1557391950354" status="failed" stop="1557391950411">
      <name>C:\Users\dongxin\PycharmProjects\5.9\scripts.testlogin.Testlogin.test_hello_002</name>
      <failure>
        <message>AssertionError: assert 0</message>
        <stack-trace>self = &lt;scripts.testlogin.Testlogin object at 0x0000015BBAFCB4E0&gt;

    def test_hello_002(self):
&gt;       assert 0
E       assert 0

scripts\testlogin.py:9: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="11308-MainThread"/>
        <label name="host" value="DESKTOP-041HO6S"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1557391950413" status="passed" stop="1557391950414">
      <name>C:\Users\dongxin\PycharmProjects\5.9\scripts.testlogin.Testlogin.test_hello_003</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="11308-MainThread"/>
        <label name="host" value="DESKTOP-041HO6S"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
